packages/auth/src/grants/model.ts

```diff
@@ -0,0 +1,54 @@
+import { Model } from 'objection'
+import { BaseModel } from '../shared/baseModel'
+import { AccessToken } from '../accessTokens/model'
+
+export enum Actions {
+  Create = 'create',
+  Read = 'read'
+}
+
+export enum StartMethods {
+  Redirect = 'redirect'
+}
+
+export enum FinishMethods {
+  Redirect = 'redirect'
+}
+
+export enum GrantState {
+  Pending = 'pending',
+  Granted = 'granted',
+  Revoked = 'revoked'
+}
+
+export class Grant extends BaseModel {
+  public static get tableName(): string {
+    return 'grants'
+  }
+
+  static relationMappings = {
+    accessTokens: {
+      relation: Model.HasManyRelation,
+      modelClass: AccessToken,
+      join: {
+        from: 'grants.id',
+        to: 'accessTokens.grantId'
+      }
+    }
+  }
+
+  public state!: GrantState
+  public type!: string
```

Comment:
Could this also be an enum?
https://github.com/interledger/rafiki/blob/main/packages/backend/src/open_payments/auth/grant.ts#L11-L16